pkg/cmd/controller/csr.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/openshift/cluster-policy-controller/pkg/operator"
+	"github.com/openshift/library-go/pkg/operator/csr"
+	"github.com/openshift/library-go/pkg/operator/events"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+)
+
+const (
+	controllerName = "csr-approver-controller"
+	// Hardcoded for the Monitoring Operator:
+	// https://github.com/openshift/cluster-monitoring-operator/blob/master/manifests/0000_50_cluster-monitoring-operator_03-role-binding.yaml#L25-L27
+	monitoringServiceAccountNamespace = "openshift-monitoring"
+	monitoringServiceAccountName      = "cluster-monitoring-operator"
+	monitoringCertificateSubject      = "/CN=system:serviceaccount:openshift-monitoring:openshift-state-metrics"
```

Comment:
as discussed in https://github.com/openshift/cluster-policy-controller/pull/61#discussion_r649790728 this should rather be `/CN=system:serviceaccount:openshift-monitoring:prometheus`

Reply:
Done

Reply:
actually, according to https://github.com/openshift/library-go/blob/master/pkg/authorization/hardcodedauthorizer/metrics.go#L23, we may want to just go with `"system:serviceaccount:openshift-monitoring:prometheus-k8s"`

Reply:
ah yes, i missed the suffix

Reply:
Done